CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## Bug Fixes
  * A warning in logs saying APM server is not available when using 1.8 with APM server 6.x
  * `ApacheHttpAsyncClientInstrumentation` matching increases startup time considerably
+ * Log correlation feature is active when `active==false`
```

Comment:
```suggestion
 * Disable log correlation feature when `active=false`
```

Reply:
The `Bug Fixes` section usually describes the bugs and not the fixes